pipl.json

```diff
@@ -8,14 +8,14 @@
     "logo_dark": "logo_pipl_dark.svg",
     "product_name": "Pipl",
     "product_version_regex": ".*",
-    "publisher": "Splunk",
+    "publisher": "Splunk Community",
     "license": "Copyright (c) 2018-2022 Splunk Inc.",
-    "app_version": "2.0.5",
+    "app_version": "3.0.0",
     "utctime_updated": "2022-01-26T23:39:50.000000Z",
     "package_name": "phantom_pipl",
     "main_module": "pipl_connector.py",
-    "min_phantom_version": "5.0.0",
-    "fips_compliant": true,
+    "min_phantom_version": "5.1.0",
+    "fips_compliant": false,
     "app_wizard_version": "1.0.0",
     "python_version": "3",
     "latest_tested_versions": [
```

Comment:
We should probably remove this as we will not be testing it.

Reply:
Done